packages/devtools_app/test/matchers/matchers.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: implementation_imports, import_of_legacy_library_into_null_safe
+// ignore_for_file: implementation_imports
```

Comment:
same question

Reply:
done in https://github.com/flutter/devtools/pull/4011